go-controller/pkg/ovn/policy.go

```diff
@@ -327,12 +327,16 @@ func getMulticastACLMatch() string {
 	return "(ip4.mcast || mldv1 || mldv2 || " + ipv6DynamicMulticastMatch + ")"
 }
 
+// Allow IGMP traffic (e.g., IGMP queries) and namespace multicast traffic
+// towards pods.
 func getMulticastACLIgrMatchV4(addrSetName string) string {
-	return "(ip4.src == $" + addrSetName + " && ip4.mcast)"
+	return "((ip4 && ip.proto == 2) || (ip4.src == $" + addrSetName + " && ip4.mcast))"
```

Comment:
I wonder if it would be a little clearer to have a constant for IGMP protocol instead of hard coding to 2.

Reply:
@vpickard You're right, and as a matter of fact, I think OVN defines a predicate "igmp" which expands to "ip4 && ip.proto == 2".  Let me try it out and I'll update the PR if it works fine.

Reply:
Done, thanks for the suggestion!